index.ts

```diff
@@ -6,7 +6,8 @@ import dotenv from "dotenv";
 dotenv.config();
 
 const app = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || '3333';
+const secret = process.env.GITHUB_SECRET || "defaultKey";
```

Comment:
please exit with an error instead of setting a known "Secret"

Reply:
created exit for "Secret"

Reply:
fixed secret